docs/index.md

```diff
@@ -0,0 +1,48 @@
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) for building VM
+images for [GitHub self-hosted runners](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners).
+
+This charm simplifies the initial deployment and building images for GitHub
+self-hosted runners.
+
+Some of the charm dependencies upgrades on a schedule to migrate security risks. The 
+landscape-client charm can be deployed with this charm to ensure other dependencies are up to date.
+
+The charm operates a set of isolated single-use OpenStack virtual machines, to build up-to-date
+images for GitHub runners.
+
+This charm will make operating GitHub self-hosted runners simple and straightforward for DevOps or
+SRE teams through Juju's clean interface.
+
+## In this documentation
+
+| | |
+|--|--|
+|  [Tutorials](https://charmhub.io/github-runner-image-builder/docs/quick-start)</br>  Get started - a hands-on introduction to using the GitHub runner charm for new users </br> | [How-to guides](https://charmhub.io/github-runner-image-builder/docs/configure-base-image) </br> Step-by-step guides covering key operations and common tasks |
```

Comment:
This file has a lot of references to `GitHub runner charm`. Is this intended?

Reply:
Yes - we are still discussing whether the documentation here is worth anything (since this charm relies heavily on GitHub runners and cannot work as a standalone charm)